src/Twee/School/Bribe Teacher.twee

```diff
@@ -24,17 +24,17 @@ She takes a quick glance at how much money you have, but just laughs coldly.
 
 You hurriedly put your wallet away.
 
-@@.teacher;"Now, $player.name, I have given a lot of thought to what you tried earlier. I am insulted you think you could buy your way out of trouble with some pocket change. 
+@@.teacher;"Now, $player.name, I have given a lot of thought to what you tried earlier. I am insulted you think you could buy your way out of trouble with some pocket change."@@
 	
-"If money is what is important to you, then perhaps I can use it to teach you a lesson. If you wish to prevent me from reporting you to the principal, it will cost you $<<print $bribeAmount>> every Monday.\
+\
 <<if $player.bribeDiscount gt 0>>\
-	Since it's already <<print window.timeCode.weekDayText()>>, I will reduce this week's amount by $<<print $player.bribeDiscount>>, to $<<print _nb>>.@@
+	@@.teacher;"If money is what is important to you, then perhaps I can use it to teach you a lesson. If you wish to prevent me from reporting you to the principal, it will cost you $<<print $bribeAmount>> every Monday. Since it's already <<print window.timeCode.weekDayText()>>, I will reduce this week's amount by $<<print $player.bribeDiscount>>, to $<<print _nb>>."@@
```

Comment:
```suggestion
	@@.teacher;Since it's already <<print window.timeCode.weekDayText()>>, I will reduce this week's amount by $<<print $player.bribeDiscount>>, to $<<print _nb>>.@@
```